iree/hal/vulkan/renderdoc_capture_manager.cc

```diff
@@ -54,7 +54,6 @@ Status RenderDocCaptureManager::Connect() {
     return NotFoundErrorBuilder(IREE_LOC)
            << "Could not load librenderdoc.so. Is it on LD_LIBRARY_PATH?";
   }
-#endif
```

Comment:
This will be improved upon in https://github.com/google/iree/pull/831. Thanks for the local fix!